appwrite/appwrite
# Fix: VCS identity collision

## What does this PR do?

Fixes a problem when personal GitHub installation could only be created on Appwrite account with same email as github email.

VCS no longer uses identities, preventing this problem. Nessessary tokens are now stored as part of installation document. These tokens are only needed when creating new repository.

## Test Plan


1. Create account by "Sign in with GitHub"

![CleanShot 2024-12-27 at 13 22 40@2x](https://github.com/user-attachments/assets/5291bf02-5e80-4bfc-af24-83ccbccf418c)

2. Sign out, and create new account `matej@appwrite.io` (different email than GitHub)

![CleanShot 2024-12-27 at 13 24 01@2x](https://github.com/user-attachments/assets/275471a1-3837-481a-bf0e-65cdd552e267)

3. Create organization, and a project. Add installation successfully

![CleanShot 2024-12-27 at 13 30 20@2x](https://github.com/user-attachments/assets/9f236a74-4c07-4775-ae7b-a1697a03b6fa)

- [x] Backwards-compatibility check:

1. Registered, made project, and linked GitHub with `1.6.x` branch
2. Switched to this branch, and created repository:

![CleanShot 2024-12-30 at 14 57 06@2x](https://github.com/user-attachments/assets/88f73175-28e6-4fa4-a6db-902c04f22c78)


3. Ensured installations don't have tokens columns, and identities do ✅

## Related PRs and Issues

x

## Checklist
